### PR TITLE
OCP 4.16 Release Notes for P/Z

### PR DESCRIPTION
**Pull request by alishaIBM**
Version(s): 4.16

Issue: https://issues.redhat.com/browse/MULTIARCH-4125

Link to docs preview:
- P: https://76393--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-16-release-notes.html#ocp-4-16-ibm-power_release-notes
- Z: https://76393--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-16-release-notes.html#ocp-4-16-ibm-z_release-notes

Draft link: https://docs.google.com/document/d/1Slyia3vRX5VVU7xO7PO9PtuEAYxmwxGsgOOLy5RcMU4/edit#heading=h.rixp64mq4cb

QE review:
For P : Manoj Kumar, Luvella , Julie
for Z : Holger Wolf, Florian Leber
- [ ] QE has approved this change.

Branch : enterprise-4.16
cc: @SNiemann15

### CODE REVIEW

File: release_notes/ocp-4-16-release-notes.adoc [hunk 1]
```diff
@@ -99,9 +99,373 @@ Warning: short name "ex" could also match lower priority resource examples.test.
...
+|Supported
+|Supported
+
+|IBM Secure Execution
+|Unsupported
+|Supported
+
+|{ibm-power-server-name} Block CSI Driver Operator
+|Supported
+|Unsupported
```
Comment (SNiemann15) [inline release_notes/ocp-4-16-release-notes.adoc:222]: @alishaIBM I think this should be moved to the Operators table

```suggestion
|{ibm-power-server-name} Block CSI Driver Operator
|Supported
|Unsupported
```

**Review by SNiemann15 — CHANGES_REQUESTED** [review]

File: release_notes/ocp-4-16-release-notes.adoc (hunk 1 above)
Comment (alishaIBM) [reply release_notes/ocp-4-16-release-notes.adoc:222]: Agree. Moved to the `Operators` section. Thanks.

File: release_notes/ocp-4-16-release-notes.adoc [hunk 2]
```diff
@@ -99,9 +99,373 @@ Warning: short name "ex" could also match lower priority resource examples.test.
...
+
+|{openshift-local-productname}
+|Unsupported
+|Unsupported
+
+|Scheduler profiles
+|Supported
+|Supported
+
+|Stream Control Transmission Protocol (SCTP)
```
[the remark this answers is not in the document]
Comment (alishaIBM) [reply release_notes/ocp-4-16-release-notes.adoc:300]: Updated the PR as per above suggestion. Thanks @SNiemann15

File: release_notes/ocp-4-16-release-notes.adoc [hunk 3]
```diff
@@ -110,9 +110,377 @@ This release introduces a new `--interactive` flag for the `oc delete` command.
...
+[discrete]
+==== {ibm-power-title} notable enhancements
+
+The {ibm-power-name} release on {product-title} {product-version} adds improvements and new capabilities to {product-title} components.
+
+This release introduces support for the following features on {ibm-power-name}:
+
+* CPU manager
+* Ingress Node Firewall Operator
+* Installer-provisioned Infrastructure for IBM Power® Virtual Server - move to CAPI
```
[the remark this answers is not in the document]
Comment (prb112) [reply release_notes/ocp-4-16-release-notes.adoc:155]: @alishaIBM I think it needs one minor update to remove `-move to CAPI` I need mjurek to confirm.

[the remark this answers is not in the document]
Comment (mjturek) [reply release_notes/ocp-4-16-release-notes.adoc:155]: I think we remove the entire IPI line, we GA'd in 4.15 and are no longer moving to CAPI

[the remark this answers is not in the document]
Comment (alishaIBM) [reply release_notes/ocp-4-16-release-notes.adoc]: Ack. Done.

**Comment by alishaIBM** [comment]
@prb112 request you to recheck the PR and confirm if the updates for IBM Power are correct. Thanks.

**Review by prb112 — APPROVED** [review]
/lgtm

File: release_notes/ocp-4-16-release-notes.adoc [hunk 4]
```diff
@@ -110,9 +110,376 @@ This release introduces a new `--interactive` flag for the `oc delete` command.
...
+
+[discrete]
+==== {ibm-z-title} and {ibm-linuxone-title} notable enhancements
+
+The {ibm-z-name} and {ibm-linuxone-name} release on {product-title} {product-version} adds improvements and new capabilities to {product-title} components and concepts.
+
+This release introduces support for the following features on {ibm-z-name} and {ibm-linuxone-name}:
+
+* Agent-based Installer ISO boot
+* Ingress Node Firewall Operator
```
Comment (SNiemann15) [inline release_notes/ocp-4-16-release-notes.adoc:130]:
```suggestion
* Ingress Node Firewall Operator
* Multi-architecture compute machines in an LPAR
```

File: release_notes/ocp-4-16-release-notes.adoc [hunk 5]
```diff
@@ -110,9 +110,376 @@ This release introduces a new `--interactive` flag for the `oc delete` command.
...
+* Secure boot
```
Comment (SNiemann15) [inline release_notes/ocp-4-16-release-notes.adoc:131]:
```suggestion
* Secure boot
* Installing on a single node in an LPAR
```

File: release_notes/ocp-4-16-release-notes.adoc (hunk 4 above)
Comment (alishaIBM) [reply release_notes/ocp-4-16-release-notes.adoc:130]: Ack. Updated the PR.

File: release_notes/ocp-4-16-release-notes.adoc [hunk 6]
```diff
@@ -110,9 +110,378 @@ This release introduces a new `--interactive` flag for the `oc delete` command.
...
+[discrete]
+==== {ibm-z-title} and {ibm-linuxone-title} notable enhancements
+
+The {ibm-z-name} and {ibm-linuxone-name} release on {product-title} {product-version} adds improvements and new capabilities to {product-title} components and concepts.
+
+This release introduces support for the following features on {ibm-z-name} and {ibm-linuxone-name}:
+
+* Agent-based Installer ISO boot
+* Ingress Node Firewall Operator
+* Installing on a single node in an LPAR
```
Comment (SNiemann15) [inline release_notes/ocp-4-16-release-notes.adoc:131]:
```suggestion

```
```suggestion
* Installing on a single node in an LPAR
```

File: release_notes/ocp-4-16-release-notes.adoc [hunk 7]
```diff
@@ -110,9 +110,378 @@ This release introduces a new `--interactive` flag for the `oc delete` command.
...
+====
+
+[discrete]
+==== {ibm-z-title} and {ibm-linuxone-title} notable enhancements
+
+The {ibm-z-name} and {ibm-linuxone-name} release on {product-title} {product-version} adds improvements and new capabilities to {product-title} components and concepts.
+
+This release introduces support for the following features on {ibm-z-name} and {ibm-linuxone-name}:
+
+* Agent-based Installer ISO boot
```
Comment (SNiemann15) [inline release_notes/ocp-4-16-release-notes.adoc:129]:
```suggestion
* Agent-based Installer ISO boot for {op-system-base} KVM
```
```suggestion
* Agent-based Installer ISO boot
```

File: release_notes/ocp-4-16-release-notes.adoc (hunk 6 above)
Comment (alishaIBM) [reply release_notes/ocp-4-16-release-notes.adoc]: Ack. Removed line 131.

File: release_notes/ocp-4-16-release-notes.adoc [hunk 8]
```diff
@@ -110,9 +110,378 @@ This release introduces a new `--interactive` flag for the `oc delete` command.
...
+* Ingress Node Firewall Operator
+* Installing on a single node in an LPAR
+* Multi-architecture compute machines in an LPAR
+* Secure boot
```
[the remark this answers is not in the document]
Comment (alishaIBM) [reply release_notes/ocp-4-16-release-notes.adoc]: Ack. Updated the PR. cc: @SNiemann15

**Comment by holgwolf** [comment]
LGTM

File: release_notes/ocp-4-16-release-notes.adoc [hunk 9]
```diff
@@ -110,9 +110,377 @@ This release introduces a new `--interactive` flag for the `oc delete` command.
 [id="ocp-4-16-ibm-z_{context}"]
 === {ibm-z-title} and {ibm-linuxone-title}
 
+With this release, {ibm-z-name} and {ibm-linuxone-name} are now compatible with {product-title} {product-version}. You can perform the installation with z/VM, LPAR, or {op-system-base-full} Kernel-based Virtual Machine (KVM). For installation instructions, see the following documentation:
```
Comment (mburke5678) [inline release_notes/ocp-4-16-release-notes.adoc:113]: You can include the link right in the sentence.

```suggestion
With this release, {ibm-z-name} and {ibm-linuxone-name} are now compatible with {product-title} {product-version}. You can perform the installation with z/VM, LPAR, or {op-system-base-full} Kernel-based Virtual Machine (KVM). For installation instructions, see xref:../installing/installing_ibm_z/preparing-to-install-on-ibm-z.adoc#preparing-to-install-on-ibm-z[Installing a cluster with on {ibm-z-title} and {ibm-linuxone-title}]
```

File: release_notes/ocp-4-16-release-notes.adoc [hunk 10]
```diff
@@ -110,9 +110,377 @@ This release introduces a new `--interactive` flag for the `oc delete` command.
...
+
+* xref:../installing/installing_ibm_z/preparing-to-install-on-ibm-z.adoc#preparing-to-install-on-ibm-z[Installing a cluster with on {ibm-z-title} and {ibm-linuxone-title}]
```
Comment (mburke5678) [inline release_notes/ocp-4-16-release-notes.adoc:115]: If you add the link to the sentence, remove this line.

File: release_notes/ocp-4-16-release-notes.adoc [hunk 11]
```diff
@@ -110,9 +110,377 @@ This release introduces a new `--interactive` flag for the `oc delete` command.
...
+
+|Stream Control Transmission Protocol (SCTP)
+|Supported
+|Supported
+
+|Support for multiple network interfaces
+|Supported
+|Supported
+
+|The openshift-install utility to support various SMT levels on {ibm-power-name} (Hardware Acceleration)
```
Comment (mburke5678) [inline release_notes/ocp-4-16-release-notes.adoc:319]:
```suggestion
|The `openshift-install` utility to support various SMT levels on {ibm-power-name} (Hardware Acceleration)
```

**Comment by mburke5678** [comment]
@alishaIBM A few comments. Otherwise LGTM

**Comment by alishaIBM** [comment]
> @alishaIBM A few comments. Otherwise LGTM

@mburke5678 thank you for the suggestions. I have updated the PR. FYI.

File: release_notes/ocp-4-16-release-notes.adoc [hunk 12]
```diff
@@ -159,9 +159,376 @@ This release introduces a new `--interactive` flag for the `oc delete` command.
 [id="ocp-4-16-ibm-z_{context}"]
 === {ibm-z-title} and {ibm-linuxone-title}
 
+With this release, {ibm-z-name} and {ibm-linuxone-name} are now compatible with {product-title} {product-version}. You can perform the installation with z/VM, LPAR, or {op-system-base-full} Kernel-based Virtual Machine (KVM). For installation instructions, see 
+xref:../installing/installing_ibm_z/preparing-to-install-on-ibm-z.adoc#preparing-to-install-on-ibm-z[Installing a cluster with on {ibm-z-title} and {ibm-linuxone-title}]
```
Comment (ShaunaDiaz) [inline release_notes/ocp-4-16-release-notes.adoc:163]:
```suggestion
xref:../installing/installing_ibm_z/preparing-to-install-on-ibm-z.adoc#preparing-to-install-on-ibm-z[Installing a cluster with on {ibm-z-title} and {ibm-linuxone-title}].
```
A complete sentence; needs a period.

**Review by ShaunaDiaz — COMMENTED** [review]
one tiny but significant nit; lmk when ready and I can merge

File: release_notes/ocp-4-16-release-notes.adoc (hunk 12 above)
Comment (alishaIBM) [reply release_notes/ocp-4-16-release-notes.adoc]: Ack. Updated the PR.

File: release_notes/ocp-4-16-release-notes.adoc [hunk 13]
```diff
@@ -222,9 +222,376 @@ This release introduces a new `--interactive` flag for the `oc delete` command.
...
+With this release, {ibm-z-name} and {ibm-linuxone-name} are now compatible with {product-title} {product-version}. You can perform the installation with z/VM, LPAR, or {op-system-base-full} Kernel-based Virtual Machine (KVM). For installation instructions, see 
+xref:../installing/installing_ibm_z/preparing-to-install-on-ibm-z.adoc#preparing-to-install-on-ibm-z[Installing a cluster with on {ibm-z-title} and {ibm-linuxone-title}].
+
+[IMPORTANT]
+====
+Compute nodes must run {op-system-first}.
+====
+
+[discrete]
+==== {ibm-z-title} and {ibm-linuxone-title} notable enhancements
```
Comment (skopacz1) [inline release_notes/ocp-4-16-release-notes.adoc:234]: Per our [docs guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#discrete-headings), even if a heading is discrete, it still needs a unique ID: 
```suggestion
[discrete]
[id="ocp-4-16-ibm-z-enhancements_{context}"]
==== {ibm-z-title} and {ibm-linuxone-title} notable enhancements
```

File: release_notes/ocp-4-16-release-notes.adoc [hunk 14]
```diff
@@ -222,9 +222,376 @@ This release introduces a new `--interactive` flag for the `oc delete` command.
...
+* xref:../installing/installing_ibm_power/installing-ibm-power.adoc#installing-ibm-power_installing-ibm-power[Installing a cluster on {ibm-power-name}]
+* xref:../installing/installing_ibm_power/installing-restricted-networks-ibm-power.adoc#installing-restricted-networks-ibm-power_installing-restricted-networks-ibm-power[Installing a cluster on {ibm-power-name} in a restricted network]
+
+[IMPORTANT]
+====
+Compute nodes must run {op-system-first}.
+====
+
+[discrete]
+==== {ibm-power-title} notable enhancements
```
Comment (skopacz1) [inline release_notes/ocp-4-16-release-notes.adoc:259]: Same comment as above:
```suggestion
[discrete]
[id="ocp-4-16-ibm-power-enhancements_{context}"]
==== {ibm-power-title} notable enhancements
```

File: release_notes/ocp-4-16-release-notes.adoc [hunk 15]
```diff
@@ -222,9 +222,376 @@ This release introduces a new `--interactive` flag for the `oc delete` command.
...
+
+The {ibm-power-name} release on {product-title} {product-version} adds improvements and new capabilities to {product-title} components.
+
+This release introduces support for the following features on {ibm-power-name}:
+
+* CPU manager
+* Ingress Node Firewall Operator
+
+[discrete]
+=== {ibm-power-title}, {ibm-z-title}, and {ibm-linuxone-title} support matrix
```
Comment (skopacz1) [inline release_notes/ocp-4-16-release-notes.adoc:269]: Same comment as above:
```suggestion
[discrete]
[id="ocp-4-16-ibm-z-power-support-matrix_{context}"]
=== {ibm-power-title}, {ibm-z-title}, and {ibm-linuxone-title} support matrix
```

**Review by skopacz1 — COMMENTED** [review]
Just left a few comments about adding IDs to headings, otherwise looks good to me! Feel free to choose different wording for your IDs, so long as they end with a {context}.

Let me know when the feedback is addressed and I can proceed to merge, thanks!

**Comment by alishaIBM** [comment]
> Just left a few comments about adding IDs to headings, otherwise looks good to me! Feel free to choose different wording for your IDs, so long as they end with a {context}.
> 
> Let me know when the feedback is addressed and I can proceed to merge, thanks!

@skopacz1 thank you for the comments. I have updated the PR accordingly and have added merge review label. Request you to take a look. Thanks.

**Comment by skopacz1** [comment]
Everything LGTM, thanks for making the changes.

**Pull request merged**